### ISSUE TITLE
Usability: Site logo links to University Homepage and Media-Faculty

### ISSUE THREAD

**Issue by jdittrich** [BauhausUniversity/MedienWiki#6]
All users tested failed to predict the link target and assumed it is linking to the Wiki startpage. So it should link to the start page.

**Comment by audiocommander** [comment]
However, there should be a link to the University's main page nevertheless…

**Comment by jdittrich** [comment]
Absolutely. But somewhere else and named "to uni-weimar.de" or some other form which is not to be mistaken. I'll suggest an appropriate position as soon as rough designs for the template are there.

**Comment by audiocommander** [comment]
No objection, just an add-on -
The University's Corporate Design must be followed:
http://www.uni-weimar.de/de/universitaet/struktur/zentrale-einrichtungen/universitaetskommunikation/corporate-design/

**Comment by jdittrich** [comment]
Thanks for the link. I read it and I think it poses no problems nor compromises. The only thing I see that could us "restrict" our design is that if we use colors, we may use the university ones and that the font needs to be Verdana – but that is no problem imho.
